jenkins-integration/jenkins-jobs/common/scripts/jenkins/pipeline.groovy

```diff
@@ -110,6 +110,19 @@ def step040_install_puppet_code(script_dir, code_deploy, server_era) {
     }
 }
 
+def step045_install_hiera_config(script_dir, code_deploy, server_era) {
+    hiera_config_source_file = (code_deploy["hiera_config_source_file"] == null) ?
+            "" : code_deploy["hiera_config_source_file"]
```

Comment:
You're right about the fact that the file existence check would need to happen on beaker, good point.  I wouldn't worry about it for now though.